src/roman_datamodels/testing/factories.py

```diff
@@ -663,18 +664,50 @@ def create_wfi_img_photom_ref(**kwargs):
     roman_datamodels.stnode.WfiImgPhotomRef
     """
     raw_dict = {
+        "F062":
+            {"photmjsr": (10 * np.random.random() * u.MJ / u.sr),
+             "uncertainty": np.random.random() * u.MJ / u.sr,
+             "pixelareasr": .2 * u.sr},
```

Comment:
This is a pixel area of 0.2 sr? I think that something like
"pixelareasr":  1.0e-13 * u.sr}
might be closer to the final values.